gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
```

Comment:
Commit message: Should be "To work around".

Reply:
Done.